index.rst

```diff
@@ -6,20 +6,423 @@
+The ion pump should be started only after ~10−6 Torr is reached (as the high pressure could destroy the ion pump).
+
+Connect the turbopump to the dewar
+-----------------------------------
+
+Put the turbopump in a stable position close enough to the dewar position (to be close to the flange connections). 
+An example setup is shown in Figure 5. 
+
+.. warning::
+   The Pfeifer Turbopump must be supported on a stable surface and strapped to the observing deck grating to ensure its safety in the case of an earthquake. 
+   The turbopump's operation is degraded if it is not operating on a secure surface.
+   A stout stool can be found on the observing floor of AuxTel which is perfect for this setup. 
+   It is likely that the instrument will need to be rotated to +20 degrees to accommodate this. 
+   The telescope mount may also need to be rotated to accommodate all of the power supplies necessary to run the turbopump and vacuum sensors.
+
+.. figure:: /_static/turbopump_setup.png
+   :name: turbopump-setup
+   :alt: Turbopump setup
+
+   Left - Connecting the turbopump to the dewar. Right - Vacuum hose connection to the dewar. 
+
+Remove the vacuum covers, and clean the turbopump and the hose flange with isoporpyl alcohol (using gloves), to avoid introducing contaminants inside the turbopump and hose.
+Connect the hose to the turbopump flange. 
+The hose has to be in a relaxed shape, with no turn or folds. 
+
+After checking the instrument dewar valve is closed, clean both the vacuum hose and instrument dewar flange with alcohol.
+Connect the other end of the hose to the dewar, without forcing the hose into a stressed position.
+This is why it may be necessary to rotate Nasmyth #2 between 0° and 20° to find a comfortable fit. 
+
+Check the turbopump output valve is closed.
+
+.. figure:: /_static/instrument_dewar_valve.png
+   :name: instrument-dewar-valve
+   :alt: LATISS dewar valve
+
+   Picture of the LATISS dewar valve. 
+
+.. note:: 
+   The turbopump pump output valve is not typically used during the procedure. 
+   What is important is that the pressure in the vacuum line to be lower than the pressure of the dewar BEFORE opening the dewar valve. 
+
+Starting the turbopump
+----------------------
+
+Connect the external pressure sensor to the turbopump. 
+This is an external sensor that is mounted on the top of the turbopump, and connects to an electronic display. 
+See figure 7.
+This pressure sensor is identical to the dewar pressure sensor on the first floor. 
+To start the electronic display read out, press and hold the up arrow key for three seconds.
+
+.. figure:: /_static/pressure_sensor.png
+   :name: pressure-sensor
+   :alt: Turbopump pressure sensor
+
+   Left – Turbopump vacuum line sensor (connected to the top of the pump). Right – Pressure electronic display.
+
 .. note::
+   The setup in the left photo of Fig. 7 is example only. 
+   Please do not operate the Turbopump on a non-rigid surface. 
```

Comment:
Remove the "Please" -- just so it's clear that it's not a request...

Reply:
👍